doc/spec_TransferElement.dox

```diff
@@ -386,10 +387,9 @@ This document describes the behaviour of the TransferElement base class, the NDR
     - 5.2.7 A write operation is executed on a transfer element that cannot be written
       - 5.2.7.1 Check with TransferElement::isWriteable() or TransferElement::isReadOnly()
       - \anchor transferElement_C_5_2_7_2 5.2.7.2 Thrown in doPreWrite() [\ref UnifiedTest_TransferElement_C_5_2_7_2 "U"]
-    - 5.2.8 Scalar, 1D or 2D registers of UserType Void
-      - The instantiation of the Void UserType is limited to the respective VoidRegister
-      - Other register types will throw an error upon instantiaion.
-    - 5.2.9 UserType Void in combination with read only and not having wait_for_new_data
+    - 5.2.8 ScalarRegisterAccessor, OneDRegisterAccessor or TwoDRegisterAccessor Instantiation with UserType ChimeraTK::Void is forbidden
```

Comment:
Again wrong wording. This bullet point is the extension of "A logic error is thrown if .. ". You cannot complete this with "instantiation is forbidden".